.vscode/LHR.code-workspace

```diff
@@ -289,9 +289,9 @@
         "recommendations": [
             "ms-vscode.cpptools-extension-pack",    // For Controls C code
             "GitHub.vscode-pull-request-github",    // For reviewing pull requests directly from VSCode
-            "xaver.clang-format",                   // For auto-formatting code on VSCode
             "eamodio.gitlens",                      // For better github stuff
-            "ritwickdey.liveserver",                // For quick viewing test reports
+            "jirkavrba.subway-surfers",             // For fun
+            "ms-vscode-remote.remote-wsl"           // For WSL
```

Comment:
```suggestion
            "ms-vscode-remote.remote-wsl"           // For WSL
            "waderyan.gitblame"                     // For tracking commit history and authors within code
```
Could we also add git blame? It's been helpful to see how recently something had been changed.